assets/css/tasks/_contact-tasks.css

```diff
@@ -0,0 +1,71 @@
+/* CONTACT PAGE */
+
+.contact-section {
+    background-image: url(/img/contact-img.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: right 10rem bottom 0;   
```

Comment:
It will look better if you set the value to "right 20rem". This is because the `.container` class has rules with margins and padding like this:
`` ''
.container {
    margin: 0-20 rem;
    lining: 7rem 40rem;
}
`` ''
So we have to adapt to their width.